test/presenters/supergroups/policy_and_engagement_test.rb

```diff
@@ -12,21 +12,177 @@
         .stubs(:fetch)
         .returns(section_tagged_content_list('case_study'))
 
-      expected = [
-        {
-          link: {
-            text: 'Tagged Content Title',
-            path: '/government/tagged/content'
-          },
-          metadata: {
-            public_updated_at: '2018-02-28T08:01:00.000+00:00',
-            organisations: 'Tagged Content Organisation',
-            document_type: 'Case study'
+      assert_equal expected_result('case_study'), policy_and_engagement_supergroup.document_list(taxon_id)
+    end
+
+    describe 'consultations' do
+      before do
+        content = content_item_for_base_path('/government/tagged/content').merge(
+          "details": {
+            "body": "",
+            "closing_date": "2017-07-10T23:45:00.000+00:00"
           }
-        }
-      ]
+        )
+
+        content_store_has_item('/government/tagged/content', content)
+      end
+
+      it 'prioritises consultations over other content' do
+        tagged_document_list = %w(
+          consultation_outcome
+          case_study
+          open_consultation
+          case_study
+          closed_consultation
+        )
+
+        expected_order = %w(
+          consultation_outcome
+          open_consultation
+          closed_consultation
+          case_study
+          case_study
+        )
+
+        MostRecentContent.any_instance
+          .stubs(:fetch)
+          .returns(tagged_content(tagged_document_list))
+
+        assert_equal expected_results(expected_order), policy_and_engagement_supergroup.document_list(taxon_id)
+      end
+
+      describe '#special_format_count' do
+        it 'only show applies special formatting to consultations' do
```

Comment:
Not sure what you're trying to say in this test name... (or the one underneath)

Reply:
@vanitabarrett I'll rename this! That's some bad English right there!